css-view-transitions-1/Overview.bs

```diff
@@ -1656,6 +1660,20 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 			1. Otherwise:
 
+				1. If any of the following is true:
+
+					- Any [=flat tree=] ancestor of |capturedElement|'s [=new element=] [=skips its contents=].
+
+					- |capturedElement|'s [=new element=] is [=element-not-rendered|not rendered=].
+
+					- |capturedElement|'s [=new element=] does not form a [=stacking context=].
```

Comment:
It does, but these conditions could also be satisfied in other ways. I don't mind requiring the view-transition-name to be the method to do this though.

Reply:
I'd like a second opinion here.

Assuming the element is rendered & not skipped, a `view-transition-name` of not-none will enforce the constraints we require. However, I chose not to use it has a proxy, because at this stage we're kinda 'done' with `view-transition-name`.

For example, once we capture the `view-transition-name` as `foo`, it creates a `foo` group. If you change the `view-transition-name` to `bar` half way through the transition, nothing happens, it's still part of the `foo` group, since that decision was made at capture time.

Because of this, it feels weird to depend on `view-transition-name` at this point of the process.

We decided against making `view-transition-name` part of the checks here. Although it would be cheap to check the `view-transition-name` hasn't changed for captured elements (and skip otherwise), it'd be more expensive to check if any new `view-transition-name`s have appear, per frame.

Reply:
I agree, the `view-transition-name` should have no bearing on the validity of the transition, outside of determining what to capture/animate at predefined stages. 

Requiring `view-transition-name` is not equivalent to requiring the three listed properties, since it also requires a fourth property: the `view-transition-name` itself. 

That being said, I'm OK with using `view-transition-name` as a proxy, but wording has to be careful here. Something like "if any of the effects that would normally be enforced by a non-`none` `view-transition-name` no longer apply"

I'm also happy with the PR as is, since it lists these out explicitly.

Reply:
I think that change is different from requiring the three conditions (stacking context, not skipped, etc). I can't find the precise resolution where we agreed what the required conditions are, and we changed them quite a bit, but I don't believe we ever had "non-none view-transition-name" as one of the required conditions

We _could_ change the set of required conditions to be "it has a view-transition-name", but maybe via an issue and not a PR

Reply:
> I can't find the precise resolution where we agreed what the required conditions are

#8139 resolved on the exact constraints and the fact that they are implicitly applied by `view-transition-name`.
#7882 resolved that if constraints on the element are unsatisfied during the animation, we should abort the transition.

The second issue wasn't explicit about which of the 2 implies unsatisfied:
- The absence of `view-transition-name`
- Not having constraints implicitly applied by the property (which could continue to apply via other properties after `view-transition-name` is removed).

Do

Reply:
> We could change the set of required conditions to be "it has a view-transition-name", but maybe via an issue and not a PR

Lets resolve on the spec text explicitly to avoid ambiguity. Filed #8548 for this.